IdentitySdkCore/IdentitySdkCore/Classes/ReachFive.swift

```diff
@@ -52,4 +53,35 @@ public class ReachFive: NSObject {
             Scope: \(scope.joined(separator: ""))
         """
     }
+    
+    public func loginCallback(tkn: String, scopes: [String]?)  -> String {
+    
+          let pkce = Pkce.generate()
+          self.storage.save(key: "PASSWORDLESS_PKCE", value: pkce)
```

Comment:
Is it possible to store the key in a variable?

Reply:
Yes possible, but it's better to store it in SecureStorage